keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -672,6 +680,12 @@ endpoints:
 The `ServiceImport.Spec.IP` (VIP) can be used to access this service from within
 this cluster.
 
+
+#### Multi-network scenario
+One or more clusters in a ClusterSet can be running on a discrete network (a non-flat network). An MCS controller can use the `network.k8s.io` `ClusterProperty` to determine if a cluster in a `ClusterSet` is running on a discrete network. Note that the endpoints of the `EndpointSlice` for a cluster on discrete network may only be representative of the pods backing the multi-cluster service and not the real pod addresses.
```

Comment:
> Note that the endpoints of the `EndpointSlice` for a cluster on discrete network may only be representative of the pods backing the multi-cluster service

How does the MCS controller know how to populate the endpoint slices with the appropriate endpoints? Does it use the ClusterProperty's `network` value? Or is that a network name that can signal to the MCS controller to use some mapping of endpoints that exists $somewhere?

Some common API here may be useful, ie: an annotation on either the `ServiceImport`, or the `ServiceExport` would allow for different services to leverage different services (may be for some QOS isolation reasons). 

The annotation could target an Ingress endpoint, public hostname, IP address, or in-cluster service.

`proxy.mcs.k8s.io: <prox-url>`

I understand that the MCS controller implementation is not part of the multi cluster API, but it is at the very least useful to think of a full end to end solution to ensure the API is robust.